kfs-core/src/main/java/edu/arizona/kfs/fp/businessobject/ErrorCertification.java

```diff
@@ -0,0 +1,67 @@
+package edu.arizona.kfs.fp.businessobject;
+
+import java.util.LinkedHashMap;
+
+import org.kuali.rice.krad.bo.PersistableBusinessObjectBase;
+
+public class ErrorCertification extends PersistableBusinessObjectBase {
+
+	private Integer errorCertID;
+	private String expenditureDescription;
+	private String expenditureProjectBenefit;
+	private String errorDescription;
+	private String errorCorrectionReason;
+	
+	public Integer getErrorCertID() {
+		return errorCertID;
+	}
+	
+	public void setErrorCertID(Integer errorCertID) {
+		this.errorCertID = errorCertID;
+	}
+	
+	public String getExpenditureDescription() {
+		return expenditureDescription;
+	}
+	
+	public void setExpenditureDescription(String expenditureDescription) {
+		this.expenditureDescription = expenditureDescription;
+	}
+	
+	public String getExpenditureProjectBenefit() {
+		return expenditureProjectBenefit;
+	}
+	
+	public void setExpenditureProjectBenefit(String expenditureProjectBenefit) {
+		this.expenditureProjectBenefit = expenditureProjectBenefit;
+	}
+	
+	public String getErrorDescription() {
+		return errorDescription;
+	}
+	
+	public void setErrorDescription(String errorDescription) {
+		this.errorDescription = errorDescription;
+	}
+	
+	public String getErrorCorrectionReason() {
+		return errorCorrectionReason;
+	}
+	
+	public void setErrorCorrectionReason(String errorCorrectionReason) {
+		this.errorCorrectionReason = errorCorrectionReason;
+	}
+	
+	protected LinkedHashMap toStringMapper() {
```

Comment:
Please parameterize all method signatures and variables; i.e., we should always use generics when possible in order to leverage preventing pernicious data structure typing bugs from ever creeping in. I understand the legacy codebase has classes implemented before generics were even in the language spec, but moving forward we should always utilize them, thus improving its robustness.